os/include/tinyara/pm/pm.h

```diff
@@ -303,9 +303,16 @@ enum pm_state_e {
 	PM_COUNT,
 };
 
-struct pm_wakeup_timer_s {
-	bool use_timer;
-	uint32_t timer_interval;
+enum pm_timer_type_e {
+	PM_WAKEUP_TIMER = 1,
+	PM_LOCK_TIMER = 2,
+	/* Scope for future expansion */
+	PM_MAX_TIMER = 8,
```

Comment:
> Usually we use a last definition of an enum to get validation of input value range
Yes. However we have not used it as a formal check anywhere in code as you correctly mentioned.

> or just showing maximum value to be added for future?
So ultimately, because no check is added currently, it is serving this purpose.

Reply:
Then, just a comment is enough? (Why is maximum 8?)

Reply:
Understood, I will modify.